stripper/ze_fragment_ops_v2_7_fix.cfg

```diff
@@ -0,0 +1,59 @@
+// prevent players from holding onto health shots when new round starts
+add:
+{
+	"classname" "game_player_equip"
+	"weapon_knife" "1"
+	"weapon_elite" "1"
+	"item_assaultsuit" "1"
+	"spawnflags" "2"
+}
+
+// fix lag caused by "missing" soundscape (temporary workaround until plugin is fixed)
```

Comment:
Nit, but a soundscape has nothing to do with this? Problem is music plugin isn't always excluding sound chars from file paths.